src/main/java/com/targomo/client/api/TravelOptions.java

```diff
@@ -1688,7 +1701,7 @@ public Geometry getIntersectionGeometry() {
 	public void setIntersectionGeometry(Geometry intersectionGeometry) {
 		this.intersectionGeometry = intersectionGeometry;
 	}
-	
+
     public boolean isDisableCache() {
         return disableCache;
     }
```

Comment:
I think you need to update the `RequestConfigurator` otherwise your change won't change anything to the request